cnames_active.js

```diff
@@ -464,7 +464,7 @@ var cnames_active = {
   "cgoatone": "cgoatone.github.io",
   "chain-able": "fluents.github.io/chain-able-site",
   "chainson": "abcdan.github.io/chainson",
-  "chatapp": "7orp3do.github.io/realtime-chat-app"
+  "chatapp": "7orp3do.github.io/realtime-chat-app",
```

Comment:
```suggestion
  "chatapp": "7orp3do.github.io/realtime-chat-app"
```
Please revert this change, so that we have a clean git blame :)